docs/recipes/github-actions.md

```diff
@@ -0,0 +1,64 @@
+
+## Environment variables
+
+The [Authentication](../usage/ci-configuration.md#authentication) environment variables can be configured with [Secret variables](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables).
+
+For this example, you'll need to setup a [`NPM_TOKEN`](https://docs.npmjs.com/creating-and-viewing-authentication-tokens) to publish your package to NPM registry and a [`GH_TOKEN`](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) to generate a release at GitHub.
+
+## Node project configuration
+
+[GitHub Actions](https://github.com/features/actions) supports [Workflows](https://help.github.com/en/articles/configuring-workflows) allowing to test on multiple Node versions and publishing a release only when all test pass.
+
+**Note**: The publish pipeline must run a [Node >= 8 version](../support/FAQ.md#why-does-semantic-release-require-node-version--83).
+
+### `.github/workflows/release.yml` configuration for Node projects
+
+This example is a minimal configuration for [`semantic-release`](https://github.com/semantic-release/semantic-release) with a build running Node 12 when receives a new commit at `master` branch. See [Configuring a workflow](https://help.github.com/en/articles/configuring-a-workflow) for additional configuration options.
+
+```yaml
+name: Release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  install:
+    name: "Generate release"
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@master
+        with:
+          node-version: '12.4.0'
+      - name: Install
+        run: npm ci
+      - name: Generate release
+        env:
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npx semantic-release
+```
+
+## Pushing `package.json` changes to `master` branch
+
+If you want to keep your `package.json` updated in your code versioning with your released version you could use [`@semantic-release/git`](https://github.com/semantic-release/git) plugin. To use it you'll need to generate a `GH_TOKEN` with [permission to push changes to `master` branch](https://help.github.com/en/articles/enabling-branch-restrictions).
+
+## Trigger semantic-release on demand
+
+There is a way to trigger semantic-relase on demand, we can just use [`repository_dispatch`](https://help.github.com/en/articles/events-that-trigger-workflows#external-events-repository_dispatch) to have control when you want to generate a release only making a HTTP request, e.g.:
+
+```yaml
+name: Release
+on: [repository_dispatch]
+jobs:
+# ...
+```
+
+So just call (with your personal `GH_TOKEN` or from a generic/ci user):
+
+```
+$ curl -v -H "Accept: application/vnd.github.everest-preview+json" -H "Authorization: token ${GH_TOKEN}" https://api.github.com/repos/[org-name-or-username]/[repository]/dispatches -d '{ "event_type": "any" }'
```

Comment:
Instead of `"event_type": "any"`, I'd suggest this configuration

```yaml
name: Release
on: 
  repository_dispatch:
    types: [semantic-release]
jobs:
# ...
```

And then make the `curl` call with `-d '{ "event_type": "semantic-release" }'`

Reply:
Oh, good point! I didn't implement in this way because doc about action says that `repository_dispatch` has no `types` an attr. But, I tested it now and works perfectly, I'll changed it. Tks!